Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Build the manager binary
+FROM golang:1.15 as builder
```

Comment:
You did upgrade for operator sdk to golang 1.16, do you want to update it here too?